src/test/java/api/support/fixtures/ItemsFixture.java

```diff
@@ -51,6 +52,16 @@ public ItemsFixture(
       contributorNameTypesClient, nameType -> getProperty(nameType, "name"));
   }
 
+  public Response updateItem(UUID itemId, JsonObject update) {
+
+    return itemsClient.attemptReplace(itemId, update);
```

Comment:
Is it important to delegate the calls instead of using `itemsClient` directly?

Reply:
Fixed, deleted methods and using itemsClient directly